stable/grafana/templates/clusterrole.yaml

```diff
@@ -15,7 +15,13 @@ metadata:
 {{- if or .Values.sidecar.dashboards.enabled .Values.sidecar.datasources.enabled }}
 rules:
 - apiGroups: [""] # "" indicates the core API group
+  {{- if eq .Values.sidecar.resourceType "both" }}
+  resources: ["configmaps", "secrets"]
```

Comment:
I understand you want to preserve backward compatibility and still support configmaps, but why even make it configurable instead of just hardcoding the "both" as the selected behavior?

Reply:
Good point. I wanted to use as little permissions as necessary. But we can go for both here, if that is not a "security concern"